features/admin-broken-link-reporting.feature

```diff
@@ -2,7 +2,7 @@ Feature: Checking a document for broken links
 
   Documents can be checked for the presence of broken links.
 
-  @javascript
+  @javascript @design-system-wip
   Scenario: checking a draft document for broken links
```

Comment:
One question on this. I think we're adding a wip to this as the JS has bot been added to load the broken links checker without reloading the page.

Do we know if we're planning on doing this or just asking the user to reload the page? And if we are planning on doing it, is there a card for it?

Reply:
Yeah, exactly!
Have added a card for it here:
https://trello.com/c/doeqw0lS/990-reintroduce-polling-ajax-for-broken-link-checker